data/dataset/redis_dataset.yml

```diff
@@ -0,0 +1,119 @@
+dataset:
+  - fides_key: fidesops_redis_dataset
+    name: Fidesops Redis Dataset
+    description: Fields stored in temporary storage to support Fidesops request execution
+    collections:
+      - name: hash_table
```

Comment:
Looking for ideas on what to call this default "collection".  As I understand it, there's no collection per se here.